pkg/apis/policy/validation/validation.go

```diff
@@ -43,14 +42,10 @@ func ValidatePodDisruptionBudget(pdb *policy.PodDisruptionBudget) field.ErrorLis
 }
 
 func ValidatePodDisruptionBudgetUpdate(pdb, oldPdb *policy.PodDisruptionBudget) field.ErrorList {
-	allErrs := field.ErrorList{}
-
 	restoreGeneration := pdb.Generation
```

Comment:
what is the purpose of setting and restoring the generation field? it seems like a no-op

Reply:
if possible, we should drop that behavior... validation shouldn't mutate objects, even temporarily